eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -328,17 +332,14 @@ jobs:
               continueOnError: true
               condition: and(succeeded(), eq(variables['VerifyChangeLogForRelease'], 'true'))
 
-        - task: PublishPipelineArtifact@1
-          inputs:
-            artifactName: packages
-            path: $(Build.ArtifactStagingDirectory)/packages
+        - template: /eng/common/pipelines/templates/steps/publish-1es-artifact.yml
+          parameters:
+            ArtifactPath: $(Build.ArtifactStagingDirectory)/packages
+            ArtifactName: packages
 
-        # After all docs artifacts are generated publish docs artifacts
-        - task: PublishPipelineArtifact@1
-          inputs:
-            artifactName: docs
-            path: $(Build.ArtifactStagingDirectory)/docs
+        - template: /eng/common/pipelines/templates/steps/publish-1es-artifact.yml
```

Comment:
Follow on for us - moving all inline publish to templateContext. Not worth holding up this PR.